### ISSUE TITLE
Twitter DM to myself can not be deleted.

### ISSUE THREAD

**Issue by wnabernik** [TwidereProject/Twidere-Android#542]
I wrote a Twitter DM to myself (as a reminder).
After trying to deleting the conversation there appairs the message "Error: Unknown error, this is propably a bug."

The conversation is shortly invisible, after refresh it appairs and repeats to send the last message.

But neither in the Twitter Webui or the Twitter Android client this conversation appears (it appears only, if i wrote a new message in this conversation, only the last part is visible in Twitter).

It won't help if i delete the conversation in Twitter, because there is then all deleted, but in Twidere the full conversation (from the beginning) is there.

It also did not help remove the account in Twidere, delete the Database & caches, delete the cache in Android Apps, and then add the account in Twidere (neither direct login or web login) - it looks good till the next autorefresh comes (in my settings all 5 minutes), then "suddenly" the whole conversation appears.

I'm using the API-Key from Twitter Android Client.
Twidere Version: 3.1.6

My temporary solution is to remove the notifications - please solve this bug.

**Comment by wnabernik** [comment]
Update: i found out, that the mobile Twitter ui/site is showing the whole conversation and is deleteable, so i did that and - after removing/adding the twitter account with all steps in between (described in the previous message) - the conversation is now really gone. As sidenote: the conversation is neither visible in the Desktop ui/site or the Twitter Android App, so if someone has the same problem, this may be a workaround (from outside Twidere, with the need of the whole cleaning-procidure).

The Problem itself (Twitter-DM to myself not deleteable, as described above) is still existing.

**Comment by mariotaku** [comment]
Should work after #709, fell free to open another issue if problem persists.
